modules/advanced_search.xq

```diff
@@ -57,15 +57,15 @@ let $query-string :=
                     "recieved-place-ids:" || $plid
                 )
             else
-                ()
-            (:, if (exists($dateFrom) and exists($dateTo)) then
-                'date-sent:["' || $dateFrom || '" TO "' || $dateTo || '"]'
+                (), 
+            if (exists($dateFrom) and exists($dateTo)) then
+                'year-sent:["' || $dateFrom || '" TO "' || $dateTo || '"]'
```

Comment:
because of the nature of my form and XHR, there always is a 'date to' and 'date from'. What I need, though, is that this query returns the items in the dates of the range + the empty dates. Something like (in pseudo code):
`
 'year-sent:["' || $dateFrom || '" TO "' || $dateTo || '"] && year-sent:['']'` <- how can I do that?
 
 ...provided that there are fields in the index with empty year-sent. There should be 10.